src/helpers/metrics.ts

```diff
@@ -0,0 +1,64 @@
+import throttle from "lodash/throttle";
```

Comment:
this is lifted from Freighter/Laboratory, which each have almost this exact code. I think it's best to keep this project in parity with our other tools.

@quietbits this is a prime example of something that can go into the shared "helpers" repo we've been talking about :D